firmware/hdmi_in.sh

```diff
@@ -22,6 +22,11 @@ FIRMWARE_DIR=$SCRIPT_DIR
 
 X=$1
 
+# parse the hex as an integer, for arithemtic purposes, then convert back to hex
```

Comment:
Maybe it's just better to do a `#define HDMI_IN_INDEX` and then calculate the frame buffer value from that?

Reply:
Yeah, that may be simpler. It'd probably have to be `HDMI_IN0_INDEX` etc.